src/ci/docker/dist-x86_64-redox/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:16.04
```

Comment:
Oh sorry I also forgot to mention, but can this be moved to the `src/ci/docker/disabled` folder?